diff --git a/arch/arm/mach-omap2/mailbox.c b/arch/arm/mach-omap2/mailbox.c
index 29b9dc3..c8fc149 100644
--- a/arch/arm/mach-omap2/mailbox.c
+++ b/arch/arm/mach-omap2/mailbox.c
@@ -289,13 +289,13 @@ static struct omap_mbox2_priv omap2_mbox_dsp_priv = {
 		.fifo_stat	= MAILBOX_FIFOSTATUS(0),
 	},
 	.rx_fifo = {
-		.msg		= MAILBOX_MESSAGE(1),
-		.msg_stat	= MAILBOX_MSGSTATUS(1),
+		.msg		= MAILBOX_MESSAGE(0),
+		.msg_stat	= MAILBOX_MSGSTATUS(0),
 	},
 	.irqenable	= MAILBOX_IRQENABLE(0),
 	.irqstatus	= MAILBOX_IRQSTATUS(0),
 	.notfull_bit	= MAILBOX_IRQ_NOTFULL(0),
-	.newmsg_bit	= MAILBOX_IRQ_NEWMSG(1),
+	.newmsg_bit	= MAILBOX_IRQ_NEWMSG(0),
 	.irqdisable	= MAILBOX_IRQENABLE(0),
 };
 
